web_readonly_bypass/static/src/js/readonly_bypass.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Allow to bypass readonly fi the value is changed
+ */
+
+openerp.readonly_bypass = function(instance) {
```

Comment:
Same here, as you rename the module you forgot to rename the object name:

```javascript
openerp.web_readonly_bypass = function(instance) {
```
otherwise the class is not loaded.